### PR TITLE
Automatically set migration_task_wait_in_seconds to double node count

### PR DESCRIPTION
**Pull request by ptnapoleon**
@mambocab to review. The relevant test now passes locally for me. I'll try it out on cassci

### CODE REVIEW

**Comment by mshuler** [comment]
+1 on code without testing it out

File: ccmlib/cluster.py [hunk 1]
```diff
@@ -305,10 +306,13 @@ def show(self, verbose):
             else:
                 node.show(only_status=True)
 
-    def start(self, no_wait=False, verbose=False, wait_for_binary_proto=False, wait_other_notice=True, jvm_args=None, profile_options=None, quiet_start=False, allow_root=False):
+    def start(self, no_wait=False, verbose=False, wait_for_binary_proto=False, wait_other_notice=True, jvm_args=None, profile_options=None, quiet_start=False, allow_root=False, set_migration_task=True):
```
Comment (mambocab) [inline ccmlib/cluster.py]: Maybe break this line for readability, as long as you're editing it?

File: ccmlib/cluster.py [hunk 2]
```diff
@@ -305,10 +306,13 @@ def show(self, verbose):
...
         if jvm_args is None:
             jvm_args = []
 
+        if set_migration_task and self.cassandra_version() >= '3.0.1':
+            jvm_args +=['-Dcassandra.migration_task_wait_in_seconds={}'.format(len(self.nodes) * 2)]
```
Comment (mambocab) [inline ccmlib/cluster.py]: Missing a space after `+=`.

**Comment by mambocab** [comment]
tested locally with:

```sh
ccm create 12-nodes -n 12 -v git:trunk
ccm start 
ps a | grep migra | less
```

I saw in each of the lines from `ps` that the migration task wait time was set to 24. I did the same with 8 nodes and saw it was set to 16. Good enough for me.

**Comment by ptnapoleon** [comment]
I need to push this down to node.start for this to work.

**Comment by ptnapoleon** [comment]
Fixes ~30 test failures on cassci. Merging.

**Pull request merged**